contrib/release.sh

```
@@ -0,0 +1,42 @@
+      echo "Unknown option: $1" >&2
+      exit 1
+  esac
+
+  shift 2
+done
+
+[ -z "$BACKEND" ] && echo 'ERROR: backend version should be provided' && exit 1
+[ -z "$FRONTEND" ] && echo 'ERROR: frontend version should be provided' && exit 1
+
+git -C api fetch
+git -C api checkout $BACKEND
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)